interventions/views.py

```diff
@@ -1,9 +1,23 @@
 from django.shortcuts import render, get_object_or_404
-
-from .models import Patient
+from django.http import HttpResponseForbidden
+from .models import Patient, Doctor
 
 
 def patient_details(request, patient_id):
     patient = get_object_or_404(Patient, id=patient_id)
     return render(request, 'interventions/patient.html', {'patient': patient})
+
+def out_patient(request, patient_id): 
+        patient = get_object_or_404(Patient,id=patient_id)
+        if request.user.Doctor:
```

Comment:
Tego ifa można by napisać inaczej, w jednej linijce. Wtedy w wypadku niepomyślnej weryfikacji mógłbyś wyrzucić error, a w przypadku pomyślnej kontynuować usuwanie i renderowanie. Podpowiedź: `if x and y`